src/components/__stories__/inquiry_artwork.tsx

```diff
@@ -0,0 +1,30 @@
+import { action, storiesOf } from "@kadira/storybook"
+import * as React from "react"
+import * as Relay from "react-relay"
+
+import Artwork from "../inquiry_artwork_brick/index"
```

Comment:
FYI There’s no need to include the `index` filename, it is a special filename that by default is the entry point of the directory it resides in.